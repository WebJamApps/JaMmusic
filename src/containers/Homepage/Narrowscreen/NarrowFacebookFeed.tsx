import React from 'react';

const FacebookFeed = (): JSX.Element => (
  <div className="notWidescreen" style={{ maxWidth: '300px', margin: 'auto' }}>
    <iframe
      className="notWideScreen"
      // eslint-disable-next-line max-len
      src="https://www.facebook.com/plugins/page.php?href=https%3A%2F%2Fwww.facebook.com%2FWebJamLLC%2F&tabs=timeline&width=300&height=500&small_header=true&adapt_container_width=true&hide_cover=false&show_facepile=true&appId=2207148322688942"
      width="300"
      height="300"
      style={{ border: 'none', overflow: 'hidden' }}
      scrolling="none"
      allow="encrypted-media"
      title="facebook ticker"
    />
  </div>
);

export default FacebookFeed;
