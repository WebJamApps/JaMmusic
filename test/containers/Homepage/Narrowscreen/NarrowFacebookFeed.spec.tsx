import React from 'react';
import { shallow } from 'enzyme';
import NarrowFacebookFeed from '../../../../src/containers/Homepage/Narrowscreen/NarrowFacebookFeed';

const wrapper = shallow<typeof NarrowFacebookFeed>(<NarrowFacebookFeed/>);

describe('NarrowFacebookFeed', () => {
  it('Renders the NarrowFacebookFeed', () => {
    expect(wrapper.find('div.notWidescreen').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
