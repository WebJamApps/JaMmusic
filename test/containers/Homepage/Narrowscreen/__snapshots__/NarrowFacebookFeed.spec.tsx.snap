// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NarrowFacebookFeed Renders the NarrowFacebookFeed 1`] = `ShallowWrapper {}`;
